ProcessLib/HT/HTFEM.h

```diff
@@ -206,18 +206,20 @@ class HTFEM : public HTLocalAssemblerInterface
         auto const& liquid_phase = medium.phase("AqueousLiquid");
 
         auto const thermal_conductivity_solid =
```

Comment:
You can drop lines 209-213, and replace lines 222-224 with MaterialPropertyLib::formEffectiveThermalConductivity with including of  "MaterialLib/MPL/Utils/FormEffectiveThermalConductivity.h" (cf. lines 348-360 in  https://github.com/ufz/ogs/blob/master/ProcessLib/ThermoHydroMechanics/ThermoHydroMechanicsFEM-impl.h#L348)

Reply:
Fixed.